chaoscenter/web/src/controllers/InviteNewMembers/InviteNewMembers.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+import { ExpandingSearchInput } from '@harnessio/uicore';
+import { useStrings } from '@strings';
+import { useGetUsersForInvitationQuery } from '@api/auth';
+import InviteNewMembersView from '@views/InviteNewMembers/InviteNewMembers';
+import { generateInviteUsersTableContent } from './helper';
+import type { InviteUserDetails } from './types';
+
+interface InviteUsersControllerProps {
+  handleClose: () => void;
+}
+
+export default function InviteUsersController({ handleClose }: InviteUsersControllerProps): React.ReactElement {
+  const { projectID } = useParams<{ projectID: string }>();
```

Comment:
```suggestion
  const { projectID } = useParams<ProjectPathParams>();
```

in routes folder make a RouteInterfaces.ts
and add 
```
export type ProjectPathParams {
projectID: string
}
```